website/src/data/gallery.json

```diff
@@ -0,0 +1,163 @@
+[
+  {
+    "title": "AutoGen Playground",
+    "link": "https://huggingface.co/spaces/thinkall/AutoGen_Playground",
+    "description": "A space to explore the capabilities of AutoGen.",
+    "image": "default.png",
+    "tags": ["web", "app"]
+  },
+  {
+    "title": "AutoGen + Semantic Kernel",
+    "link": "https://devblogs.microsoft.com/semantic-kernel/autogen-agents-meet-semantic-kernel/",
+    "description": "A discussion on how AutoGen agents integrate with Semantic Kernel.",
+    "image": "default.png",
+    "tags": ["tools", "app"]
+  },
```

Comment:
Shouldn't this be in the "Ecosystem" page?

Reply:
@skzhang1

Reply:
resolved.